www/src/data/sidebars/doc-links.yaml

```diff
@@ -6,14 +6,33 @@
       link: /docs/
     - title: Quick Start
       link: /docs/quick-start/
-    - title: Recipes
-      link: /docs/recipes/
     - title: Plugin Library
       link: /plugins/
     - title: Starter Library
       link: /starters/
     - title: Awesome Gatsby Resources
       link: /docs/awesome-gatsby-resources/
+    - title: Recipes
```

Comment:
This makes sense to me, since the items not grouped in directories kinda mess with the appearance of the toggles to make you think items are related when they aren't. I've brought the arrangement of items up internally to see if there are more sweeping changes we can make, but this seems good for now.